crates/turborepo-lib/src/process/child.rs

```diff
@@ -92,6 +92,7 @@ enum ChildHandleImpl {
 }
 
 impl ChildHandle {
+    #[tracing::instrument(skip(command))]
```

Comment:
Does this directive add to tracing or skip it from tracing?

Reply:
It enables tracing for the function call, but removes the `command` arg from showing up as a field in the tracing span.